src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -73,29 +80,29 @@ if (!file_exists("$PHPUNIT_DIR/phpunit-$PHPUNIT_VERSION/phpunit") || md5_file(__
     @mkdir($PHPUNIT_DIR, 0777, true);
     chdir($PHPUNIT_DIR);
     if (file_exists("phpunit-$PHPUNIT_VERSION")) {
-        passthru(sprintf('\\' === DIRECTORY_SEPARATOR ? 'rmdir /S /Q %s > NUL': 'rm -rf %s', "phpunit-$PHPUNIT_VERSION.old"));
+        $passthru_or_fail(sprintf('\\' === DIRECTORY_SEPARATOR ? 'rmdir /S /Q %s > NUL': 'rm -rf %s', "phpunit-$PHPUNIT_VERSION.old"));
```

Comment:
this can and should fail silently by design, isn't it?

Reply:
What is it trying to do here at all?

remove the `-.old` version if it is there, then rename `phpunit-$VERSION` to `phpunit-$VERSION.old`, and then try removing that again?

Doesn't that effectively remove both the `.old` and current one?